src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        Color[] color = Color.values();
```

Comment:
It's better to use the method name() for getting the String representation of enum constants instead of relying on toString().